app/assets/javascripts/oxalis/view/right-menu/dataset_info_tab_view.js

```diff
@@ -221,6 +225,41 @@ class DatasetInfoTabView extends React.PureComponent<DatasetInfoTabProps> {
     );
   }
 
+  handleConvertToHybrid = async () => {
+    await Model.save();
+    await convertToHybridTracing(this.props.tracing.annotationId);
+    location.reload();
+  };
+
+  getTracingType(isPublicViewMode: boolean) {
+    if (isPublicViewMode) return null;
```

Comment:
I know that the variable `isPublicViewMode` existed before, but I think we should rename this. This variable is only true iff you click "view" on a dataset, right? Or is it also true if you visit "foreign" tracing? And how does "public" come into play? I'd rename (and maybe invert the semantics) it to "isTraceMode" or "isTracing" or "isDatasetViewMode" or just "isViewMode".
If the variable is also true when displaying foreign tracings, I'd only hide the "convert" button in this case (which is probably already guarded due to the allowUpdate attribute). So, you can still see the tracing type when viewing a foreign tracing.